src/ethereum_test_tools/exceptions/evmone_exceptions.py

```diff
@@ -77,6 +77,9 @@ class EvmoneExceptionMapper:
         ExceptionMessage(
             EOFException.CONTAINER_SIZE_ABOVE_LIMIT, "err: container_size_above_limit"
         ),
+        ExceptionMessage(
+            EOFException.INCOMPATIBLE_CONTAINER_KIND, "err: incompatible_container_kind"
```

Comment:
Seems to me that `INCOMPATIBLE_CONTAINER_KIND` is something we should not see in any tests unless the test is aimed to, e.g. test a eof create tx that contains a runtime container, but I think that is not the case for any of the tests that we currently have.

Also yes I think this is a duplicate of #661, let me move your fix of `src/ethereum_test_tools/spec/eof/eof_test.py` to that branch and should we close this one?

Reply:
Yes, sure.

Reply:
Only this addition is left. The tests anticipated this error, it just wasn't implemented in evmone at the time tests were created.